Dangerfile

```diff
@@ -19,3 +19,5 @@ no_changelog_entry = !git.modified_files.include?("CHANGELOG.md")
 if has_app_changes && no_changelog_entry && not_declared_trivial
   fail("Any changes to library code need a summary in the Changelog.")
 end
+
+jazzy.check fail: :modified
```

Comment:
I tried with warn first, it posted both failure and warning messages, so I changed it to fail. I think just `modified` should be enough. If we would want for any reason to keep something undocumented, other PRs should not be affected by that. And if everything is documented there will be no problem at all.